server/app/routes/api/orders.js

```diff
@@ -6,6 +6,15 @@ require('../../../db/models/order');
 
 var Order = mongoose.model('Order');
 
+router.param('id', function (req, res, next, id) {
```

Comment:
want to change id to orderId to keep consistent with wine, user, and review routes?